### ISSUE TITLE
Bug: Can't open items when search is started from Columns View

### ISSUE THREAD

**Issue by Lukiluc29** [files-community/Files#13311]
### Description

I've encountered an issue where, when I start a search for an item, once the search is complete, I am unable to open any of the items.

### Steps To Reproduce

1. Set folder location to use Columns View
2. Initiate a search
3. The results will be displayed using the Details View (as expected) but double clicking an item won't do anything

### Requirements

Investigate and fix any issues preventing the opening of search results.

### Files Version

5.5.25.0

### Windows Version

22H2 22621.1992

### Log File

![Capture d'écran 2023-09-01 180611](https://github.com/files-community/Files/assets/128183422/7ed5e669-a61b-4c38-889f-9f7058403a2b)
![Capture d'écran 2023-09-01 180955](https://github.com/files-community/Files/assets/128183422/88b187f6-c4fe-4950-bf4a-11985c90d9b9)

**Comment by yaira2** [comment]
> Identify the cause of the layout switch during searches

The layout change is intentional for search results, as for opening items, this looks like a bug.

**Comment by Lukiluc29** [comment]
> The layout change is intentional for search results, as for opening items, this looks like a bug.

I also thought it must have been intentional. I'm going to remove it.

**Comment by yaira2** [comment]
Is this still an issue with #13307?

**Comment by Lukiluc29** [comment]
Sorry for the delayed response. This is still an issue on my end.
![Capture d'écran 2023-09-27 183531](https://github.com/files-community/Files/assets/128183422/005b1a29-87ec-4078-8c5f-a5ccd16c17b0)

**Comment by yaira2** [comment]
@Lukiluc29 I'm having trouble reproducing this one, can you share a screen recording?

**Comment by Lukiluc29** [comment]
I just ran some tests on my side, but it turns out it's when I initiate a search while in column mode.

**Comment by yaira2** [comment]
Does this affect files and folders?

**Comment by Lukiluc29** [comment]
> Does this affect files and folders?

No, it doesn't affect anything; it's just that there's no option to open a file from the search.

**Comment by yaira2** [comment]
What about folders?

**Comment by Lukiluc29** [comment]
I wanted to write elements

**Comment by yaira2** [comment]
Can you update the steps to repro?

**Comment by yaira2** [comment]
Related to #14058

**Comment by BernardoSousa03** [comment]
Hello @yaira2, i'm trying to figure out what is the source of this problem and found out that the CommandManager.ActionCommand.IsExecutable is set to False. In other cases where the bug doesn't happen, it's set to True, letting the code open the element (file/folder). Could you give me some guidance as to find out why it's set to False? Also it's set to false for every single element. When is CommandManager created/updated?

**Comment by yaira2** [comment]
`IsExecutable` is set by each action separately.

**Comment by Lukiluc29** [comment]
> Related to #14058

It's the same problem

**Comment by yair100** [comment]
@Lukiluc29 this issue should be resolved in `main`.

**Comment by Lukiluc29** [comment]
It's still not resolved. Unless I start the search again.

https://drive.google.com/file/d/1Wfnw-KU-LHOU5qZryUQvBxZeQ8Mh6_oq/view?usp=drive_link

**Comment by lucas-thiebaut** [comment]
@yaira2 It's still not resolved. Unless I start the search again.

https://drive.google.com/file/d/1Wfnw-KU-LHOU5qZryUQvBxZeQ8Mh6_oq/view?usp=drive_link

**Comment by yair100** [comment]
Thank you for the report.

**Comment by yair100** [comment]
@lucas-thiebaut is this still an issue?

**Comment by lucas-thiebaut** [comment]
> [@lucas-thiebaut](https://github.com/lucas-thiebaut) is this still an issue?

I've performed further tests and everything seems to be working correctly on my end. Therefore, I'm closing this ticket.
